src/rulesets/oas/functions/typedEnum.ts

```diff
@@ -1,7 +1,9 @@
-import type { IFunction, IFunctionContext } from '../../../types';
+/* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment */
```

Comment:
The rule doesn't quite like `@stoplight/spectral-functions` for some reason :man_shrugging:
Had worked perfectly prior to the addition of aliases